selfservice/strategy/link/strategy_recovery_test.go

```diff
@@ -159,18 +165,16 @@ func TestAdminStrategy(t *testing.T) {
 
 		require.Len(t, f.UI.Messages, 1)
 		assert.Equal(t, "You successfully recovered your account. Please change your password or set up an alternative login method (e.g. social sign in) within the next 60.00 minutes.", f.UI.Messages[0].Text)
+
+		addr, err := reg.IdentityPool().FindVerifiableAddressByValue(context.Background(), identity.VerifiableAddressTypeEmail, "recoverme@ory.sh")
+		assert.NoError(t, err)
+		assert.False(t, addr.Verified)
```

Comment:
Shouldn't this be true now?

Reply:
No. This test is about the creation of a recovery link by an admin. So it is not sent to the user's eMail. This way the email address is not verified.

Reply:
Makes sense!